openff/bespokefit/executor/services/fragmenter/cache.py

```diff
@@ -27,7 +31,7 @@ def cached_fragmentation_task(
 
     task_id = worker.fragment.delay(
         cmiles=task.cmiles,
-        fragmenter_json=task.fragmenter.json(),
+        fragmenter_json=fragment_string,
         target_bond_smarts=task.target_bond_smarts,
```

Comment:
```suggestion
        target_bond_smarts=target_bonds_string,
```

Reply:
I am not sure this works if `target_bonds_string` is `null` or should I be using json loads and dumps here?